book/art/voting.bob

```diff
@@ -0,0 +1,16 @@
+                 .------------.        .------------.     .------------.     .------------.
+                 |   Staker   |        |  Network   |     | VoteSigner |     |  Validator |
```

Comment:
That box labeled Network is the Cluster.  The network is the thing that connects all those nodes.